src/Controller/DocumentController.php

```diff
@@ -159,28 +160,39 @@ public function edit(CaseEntity $case, Document $document, Request $request): Re
     }
 
     /**
-     * @Route("/{document_id}", name="document_delete", methods={"DELETE"})
+     * @Route("/{document_id}", name="document_delete", methods={"POST"})
```

Comment:
`methods` should allow both `GET` (form rendering the form) and `DELETE` for actually deleting the agenda.